src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -151,6 +154,7 @@ const Byline = ({ blocks, children }: PropsWithChildren<Props>) => {
               <strong
                 css={[
                   BylineCss.author,
+                  BylineCss.authorLink,
                   getSansBold(service),
                   getBodyCopy(script),
```

Comment:
Should we getting the typography from the theme now? Can do this in another PR though

Reply:
Yep, we could use the theme now. I'll write up a seperate ticket for this.